doc/api/http2.md

```diff
@@ -3137,4 +3163,5 @@ following additional properties:
 [`tls.TLSSocket`]: tls.html#tls_class_tls_tlssocket
 [`tls.connect()`]: tls.html#tls_tls_connect_options_callback
 [`tls.createServer()`]: tls.html#tls_tls_createserver_options_secureconnectionlistener
+[`tls.Server.close()`]: tls.html#tls_server_close_callback
```

Comment:
A nit: it seems this need to go before `` [`tls.TLSSocket`]: ``, these references are sorted in ASCII order.

Reply:
Whoops, yep I can sort.